### PR TITLE
Change alert style

### PR DESCRIPTION
**Pull request by journey2w**
### Change Description ###

New Tizen custom markdowns were added so I changed the alert style as the new style.

- Reference: PR #989

### CODE REVIEW

File: docs/application/web/get-started/wearable-watch/first-app-watch.md [hunk 1]
```diff
@@ -162,7 +159,7 @@ Pay attention to the following main issues in the application source code, to un
     </widget>
     ```
 
-    > **Note**
+    > [!NOTE]
     >
```
Comment (mijin85cho) [inline docs/application/web/get-started/wearable-watch/first-app-watch.md]: It looks like this additional > doesn't affect the layout of Note, but would you please check it again?

image

File: docs/application/web/get-started/wearable-widget/first-app-widget.md [hunk 2]
```diff
@@ -422,8 +416,7 @@ To run the application on the target device:
 
         Like with the [emulator](#viewer), you can add the widget to the home screen to easily access it.
 
-    > **Note**
-    >
+    > [!NOTE]
     > The application is launched using the default debug run configuration. To create and use another configuration:
     > 1.  In the `Project Explorer` view, right-click the project and select `Run As > Run Configurations`.
```
Comment (mijin85cho) [inline docs/application/web/get-started/wearable-widget/first-app-widget.md]: Just a question.

Is there any reason that these sentences in note looks in green?

image

File: docs/application/web/get-started/tv/first-app.md [hunk 3]
```diff
@@ -307,8 +305,7 @@ To run the application on a target device:
 
    3. Confirm that the application launches on the target device.
 
-   > **Note**
-   >
+   > [!NOTE]
```
Comment (mijin85cho) [inline docs/application/web/get-started/tv/first-app.md]: It looks like this note needs a line break.
image

File: docs/application/web/get-started/tv/first-samsung-tv-app.md [hunk 4]
```diff
@@ -303,8 +301,7 @@ To run the application on a target device:
 
     3. Confirm that the application launches on the target device.
 
-    > **Note**
-    >
+    > [!NOTE]
```
Comment (mijin85cho) [inline docs/application/web/get-started/tv/first-samsung-tv-app.md]: It looks like this note needs a line break.

image

File: docs/application/web/get-started/mobile/first-app.md [hunk 5]
```diff
@@ -228,8 +226,7 @@ To run the application on the emulator:
 
         ![Application running in the emulator](media/emulator_running_mw.png)
 
-        > **Note**
-        >
+        > [!NOTE]
         > If the emulator display has switched off, you cannot see the application launch. To see the application on the emulator screen:
```
Comment (mijin85cho) [inline docs/application/web/get-started/mobile/first-app.md]: It looks like the list items in the note are in green, the same color with the title, "Note".
Is it intended?

image

File: docs/application/web/get-started/tv/first-samsung-tv-app.md (hunk 4 above)
Comment (journey2w) [reply docs/application/web/get-started/tv/first-samsung-tv-app.md]: Please don't leave a comment here, if an issue is not related to contents but raise an issue on https://github.sec.samsung.net/RS8-TIZENWEBSITE/docs.tizen.org/issues. I will do it this time.

Reference: https://github.sec.samsung.net/RS8-TIZENWEBSITE/docs.tizen.org/issues/246

**Pull request merged**